yacman/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.0"
+__version__ = "0.4"
```

Comment:
can we just always include the patch decimal (so `0.4.0`) just so all versions are the same format?m I'm trying to standardize that so we're always compliant with semver.